src/vscodeUtils.ts

```diff
@@ -25,7 +25,32 @@ export function getWorkspaceFolder(folderName: string): vscode.WorkspaceFolder |
 // because it also contains Output panes which are considered editors
 // see also https://github.com/microsoft/vscode/issues/58869
 export function getActiveTextEditor(): vscode.TextEditor | undefined {
-  return vscode.window.visibleTextEditors.find((textEditor) => !!textEditor.viewColumn);
+  const activeEditor = vscode.window.visibleTextEditors.find((textEditor) => !!textEditor.viewColumn);
```

Comment:
What's the reason to extract this into a variable before returning?

Reply:
Was part of debugging, I can revert back